app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema")
+        self.table_name = "actor"
+
+    def all(self) -> list:
+        select_all = self._connection.execute(
```

Comment:
`selct_all` is a bad variable name because it's an action but the variable name should display what is stored in it